dex-it/src/test/scala/com/wavesplatform/it/sync/smartcontracts/OrdersFromScriptedAccTestSuite.scala

```diff
@@ -34,6 +34,8 @@ class OrdersFromScriptedAccTestSuite extends MatcherSuiteBase {
   private val aliceAsset = IssuedAsset(aliceAssetTx.id())
   private val aliceWavesPair = AssetPair(aliceAsset, Waves)
 
+  private val orderV2 = mkOrder(bob, aliceWavesPair, OrderType.BUY, 500, 2.waves * Order.PriceConstant, smartTradeFee, version = 2)
```

Comment:
`def`, otherwise you create the one order and use it everywhere

Reply:
fixed